scripts/install.sh

```diff
@@ -195,6 +195,12 @@ uninstall() {
     del_svc "${SVC_BPFD_SVC}"
     del_bin "${BIN_BPFD}"
     del_bin "${BIN_BPFCTL}"
+
+    # Delete clsact and ingress qdiscs, in the default namespace and all network
+    # namespaces.  This has the side-effect of removing any tc_dispatchers (or
+    # other programs) that happen to be attached to them.
+    echo "Delete qdiscs:"
+    delete_all_bpfman_qdiscs
```

Comment:
what if we didn't create those qdiscs but something else did?

Reply:
This is from a script I had to clean up the tc dispatchers if something went wrong and it left them hanging.  They also get left hanging if someone simply calls uninstall when they exist.  However, as you probably noted, it does wipe out those qdiscs, and everything on them, so it's more destructive than it needs to be.  I think this is fine for development, or the case where we know we're the only users of the qdiscs; however, maybe not a default part of uninstall.  With a little work I think I can just detach the programs attached to the priority we use.  That's still not perfect, so with some more work I can probably figure out how to just detach any tc dispatchers that get left hanging.

This change is independent from this pr, so I can either leave it as is (it probably helps more than it hurts right now), or back it out.  In either case, we should still improve it to just detach the hanging tc dispatchers. Let me know.

Reply:
It's fine, just wanted to confirm that it's only developer-facing and we're not requiring users to run it.